python/lsst/alert/packet/schemaRegistry.py

```diff
@@ -122,13 +134,13 @@ def calculate_id(schema):
         schema_id : `int`
             The calculated ID.
         """
-        # Significant risk of collisions with more than a few schemas;
-        # CRC32 is ok for prototyping but isn't sensible in production.
-        return zlib.crc32(json.dumps(schema.definition,
-                                     sort_keys=True).encode('utf-8'))
+        numbers = re.findall(r'\d+', schema.definition['name'])
```

Comment:
Why doesn't this just call the method `schema.get_schema_id()` you've defined above?  That way you're not duplicating code. And you could probably deprecate `calculate_id` entirely.

Reply:
Probably because I made this one before the other. I've removed it entirely now since your right with the one you don't need a second.